cmd/libsnap-confine-private/utils.h

```diff
@@ -39,6 +39,48 @@ bool sc_is_debug_enabled(void);
  **/
 bool sc_is_reexec_enabled(void);
 
+/**
+ * sc_identity describes the user performing certain operation.
+ *
+ * UID and GID may represent actual user and group accounts. Either value may
+ * also be set to -1 to indicate that a specific identity change should not
```

Comment:
Not sure you can use -1 here, IIRC uid_t and gid_t are unsigned. Also, don't see any code to handle the special cases referenced here, suggesting the comment needs to be updated.

Reply:
That's a stale comment. I'll adjust it.

Reply:
Done